### PR TITLE
Release Notes - 1.5.10 -draft 1

### PR DESCRIPTION
**Pull request by darksidemilk**

### CODE REVIEW

File: Release Notes.MD [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+* Windows 11 compatibility added to notes and docs
+* Updated Readme for various things
+* Updated contribution guidelines
+* Update udpcast to version 20200328
+* Improved client downloader in web gui
+* Repair host listing icons for Linux systems
+* Try to find information of machines currently reporting empty OS name
+* Allow for replication of sub directories, e.g. /images/drivers/video.… 
+* Add kernel parameter for udpcast maximum wait time
+* Improve groumanager.log output a fair bit
```
Comment (lukebarone) [inline Release Notes.MD:29]: I assume this should be `groupmanager.log`?

File: Release Notes.MD [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+* Various php 8 related fixes
+* Fix incorrect array index name in location plugin code.
+* Do not install sysv-rc-conf on Debian or Ubuntu, thanks @stevemoca
+* Allow -U parameter to work as expected in installer, thanks to @lukebarone 
+* Prevent from installing package dhcp* when not selected by installer
+* Fixed behavioral issues of replication services
+* Prevent from saving the whole snapin object when only associations need saving
+* Silence warnings when upgrading a FOG install with services being enabled
+* Remove inconsistent excess kernel reporting imformation, thanks @wayneworkman
```
Comment (lukebarone) [inline Release Notes.MD:56]: Should be `information`

File: Release Notes.MD [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+* Make MulticastTask getAllMulticastTasks() function non static as call 
+* Fixes for several parts of the iPXE boot menu like delete and approve hosts
+* Repair installer for RedHat, Alma, Rocky and CentOS
+* Fix snapin deletion causing an undefined variable exception with PHP 8
+* Fix display of name for capone plugin within FOG configuration page
+* Fix issue where filename got corrupted when uploading a new snapin file
+* Fix bug in iPXE embedded scripts which prevents drive boot method EXIT 
+* Prevent AD fields to be overwritten with default settings even if fie…
```
Comment (lukebarone) [inline Release Notes.MD:64]: I think the comment got cut off here

File: Release Notes.MD (hunk 1 above)
Comment (darksidemilk) [reply Release Notes.MD:29]: Good catch
Does hitting resolve add this change to the pull request?

File: Release Notes.MD (hunk 3 above)
Comment (darksidemilk) [reply Release Notes.MD:64]:
```suggestion
* Prevent AD fields from being overwritten with default settings even if fields are not empty 
```

File: Release Notes.MD [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+We have squashed bugs, further hardened the security of the system, and introduced new features, improvements, and support for new operating systems.
+
+Some highlights include
+
+* PHP 8 support (partial)
+* Windows 11 support
+* More customizable printer configurations
+* More security checks to prevent users from accessing administrative functions accidentally or maliciously
+* Added and improved pxe boot menu options
+* Support for recent Linux OS's and an improved installer experience
```
Comment (lukebarone) [inline Release Notes.MD:15]: OSes. OS's implies possession.

File: Release Notes.MD [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+To view the full list of commits with the code changes introducing these fixes and features, see the [dev branch commit history](https://github.com/FOGProject/fogproject/commits/dev-branch)
+
+### Security Fixes
+
+* Add authentication check in getfiles to prevent file enumeration
+* Add security checks to default pxe boot options like delete, change windows key, and join multicast domain
+* Do not prepend filename to kernel information, thanks @wayneworkman
+* Remove replication of ssl files and directories
+* Update select JS and CSS library to latest version 4.0.13
+* Prevent leak of AD password when applying default AD infos in host
```
Comment (lukebarone) [inline Release Notes.MD:33]: infos -> information

File: Release Notes.MD [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+* Remove header not posing much of security as mentioned in #473, thanks @djohle
+* Add query for >=MariaDB 10.4 mysql.global_privs table
+* Check download URL and destination filename to prevent from maliciou use, thanks to sml for reporting
+* Added web login logging
+* Add changes to firewall and SELinux checks
+* Added a defined [security policy](https://github.com/FOGProject/fogproject/blob/8ce090ba82ce22208daa0a20bc5018364e20949a/SECURITY.md)
+
+### improvements
```
Comment (lukebarone) [inline Release Notes.MD:41]: Capital "I"

File: Release Notes.MD [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+* Switch default database engine from MyISAM to InnoDB
+* Point FOG web UI kernel update page to the kernel repo on github.
+* Switch to snponly.efi for UEFI systems as default
+* Save resouces by only enabling systemd tftp.socket
+* Move from xinetd to systemd for TFTP connection handling on redhat
+* Change internet connection check to better suit corporate networks
+* Add hint on image data files not being deleted through multi delete
+* Updated Internationlization support (multiple languages)
+* Installer support for more operating systems
+* Attempt https and http for wakeOnLAN
```
Comment (lukebarone) [inline Release Notes.MD:74]: WakeOnLAN

File: Release Notes.MD [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+* Separate the scheduled tasks from the pm tasks
```
Comment (lukebarone) [inline Release Notes.MD:75]: pm - I assume Power Management?

File: Release Notes.MD [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+* More verbose Logging in various locations
+* Installer pre-rew check improvements
```
Comment (lukebarone) [inline Release Notes.MD:77]: I assume pre-requisite?

File: Release Notes.MD [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+* [fog zazzles commit history](https://github.com/FOGProject/zazzles/commits/master)
+
+### Security Fixes
+
+* Add a break to logging and authentication attempts when the fog client fails to authenticate after 5 attempts, thanks to @Iri5s
+* Update external library SharpZipLib to v1.3.3 to fix security issues
+* Update external library Newtonsoft.Json to v13.0.1
+* Allow using certificate files from local filesystem
+* Update timestamp server list and use new signing cert
+* Run authentity check in mac and linux os clients
```
Comment (lukebarone) [inline Release Notes.MD:128]: authenticity; Mac OS; Linux OS

File: Release Notes.MD [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+* Added ability to use custom PrintUI parameters, thanks to @Iri5s
+* Update Zazzles library to the latest version build
+* Updated internationalization support
+* Update copyright information in all source files and assembly to 2022
+* Change logging of the installer for a more clear output on the console 
+* Add XML schema that came with Quartz.NET last version update
+
+### Bug Fixes
+
+* Fix applying of printer settings through PrintUI, thanks tatanas.
```
Comment (lukebarone) [inline Release Notes.MD:142]: Shoudl tatanas be @'d

**Comment by Neustradamus** [comment]
Maybe better to have only one commit?

**Comment by Sebastian-Roth** [comment]
@darksidemilk Would you find the time to update the PR with @lukebarone's suggestions? I will squash and merge it then.

**Comment by lukebarone** [comment]
Looks great! Thanks for the review!

**Comment by Sebastian-Roth** [comment]
@darksidemilk Thanks heaps for compiling this extensive list and @lukebarone for reviewing!

**Comment by darksidemilk** [comment]
I just noticed that I listed fos and the web ui in the first heading and then listed fos again. Oops

**Comment by Sebastian-Roth** [comment]
Thanks, fixed in current dev-branch. As well I started to add older release note information: 67e408651

**Pull request merged**